scripts/aks-as-mgmt.sh

```diff
@@ -260,4 +260,4 @@ rm -r combined_contexts.yaml
 rm -f tilt-settings-temp.yaml
 }
 
-main
+main
```

Comment:
Could you add back the trailing newline? That seems to be preferred in this codebase.